src/test/java/SeleniumFramework.java

```diff
@@ -1,51 +1,72 @@
-import org.junit.Test;
+import org.testng.annotations.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.Assert;
 
-import java.sql.Time;
-import java.util.concurrent.TimeUnit;
 
 public class SeleniumFramework {
 
     @Test
-    public void passwordCheck() {
+    public static void LoginSuccess() throws InterruptedException {
         WebDriver driver = new ChromeDriver();
-        driver.get("https://deens-master.now.sh/login");
+        driver.get("https://deens.com/?internaltraffic");
+        WebElement login_button = driver.findElement(By.cssSelector("a[href*='login']"));
+        login_button.click();
+        login_button.sendKeys("Mendygax@gmail.com");
+        Thread.sleep(1000);
+
+        WebElement password = driver.findElement(By.cssSelector("#password"));
+        password.click();
+        password.sendKeys("Positive89");
+
+        Thread.sleep(1000);
+
+        driver.findElement(By.cssSelector("[data-testid=\'loginSubmit\']")).click();
+
+        Thread.sleep(2000);
+        driver.quit();
+    }
+
+//    TC2 Login Wrong credentials (not existing email)
+    @Test
+    public static void WrongEmail() throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://deens.com/?internaltraffic");
```

Comment:
Please use maximaze method for driver window or handle this element which I mentioned above